content/collections/source-catalog/en/ga4.md

```diff
@@ -0,0 +1,150 @@
+integration_category:
+  - data-warehouse-data-lake
+integration_type:
+  - group-properties
+  - user-properties
+  - raw-events
+title: 'Google Analytics 4 Import'
+source: 'https://www.docs.developers.amplitude.com/data/sources/ga4/'
+category: Warehouse
+connection: source
+partner_maintained: false
+integration_icon: partner-icons/ga4.svg
+exclude_from_sitemap: false
+---
+With Amplitude's Google Analytics 4 integration, you can ingest Google Analutics 4 data into your Amplitude project.
+
+## Prerequisites
+
+To get started with import from GA4, you need to take care of a few prerequisites.
+
+* [Set up a BigQuery Link.](https://support.google.com/analytics/answer/9823238?hl=en) 
+* [Create a GCS bucket.](https://cloud.google.com/storage/docs/creating-buckets) Amplitude recommends one dedicated to this purpose. The ingestion process must offload data to a GCS bucket before ingesting it into Amplitude. This is due to BigQuery's limited export options.
+* [Create a Service Account](https://cloud.google.com/iam/docs/service-accounts-create#console) with permissions granted for the bucket and tables you want to ingest, then [get the service account key](https://cloud.google.com/iam/docs/keys-create-delete). The Service Account must have the following roles granted to it:
+  * **BigQuery**
+    * BigQuery Job User at the project level.
+    * BigQuery Data Viewer at the resource level necessary to access your data to be ingested. If your data is in a single table, just grant the service account BigQuery Data Viewer for that table resource. If the query requires multiple tables or datasets, grant BigQuery Data Viewer on all tables or datasets in the query.
+  * **Cloud Storage**
+    * Storage Admin on the GCS bucket you're using for ingestion.
+* Depending on your company's network policy, you may need to add the following IP addresses to your allowlist to allow Amplitude's servers to access your BigQuery instance:
+  * Amplitude US IP addresses:
+    * 52.33.3.219
+    * 35.162.216.242
+    * 52.27.10.221
+  * Amplitude EU IP addresses:
+    * 3.124.22.25
+    * 18.157.59.125
+    * 18.192.47.195
+
+## How Amplitude imports your GA4 data
+
+Here’s a step-by-step breakdown of how Amplitude imports your GA4 data:
+
+1. Export GA4 Data to BigQuery
+
+    First, your GA4 data is exported to your BigQuery project through BigQuery Linking. This process sets up an automated flow of event data from GA4 to BigQuery.
+
+2. Unload Data from BigQuery to Your GCS Bucket
+
+    To handle BigQuery’s export limitations (e.g., export size restrictions), we unload your data from BigQuery to the GCS bucket you provided. This ensures data readiness for ingestion into Amplitude.
+
+    * Delay Handling:
+
+      To ensure your data is complete, we account for [delays caused by GA4’s export process](https://support.google.com/analytics/answer/9358801?hl=en):
+      * 72 hours for daily exports.
+      * 24 hours for streaming exports.
+
+    * Catch-up Importing:
```

Comment:
would be helpful to also describe how initial batch works, and how next batch imports to ensure all data will be imported.